python/rpdk/java/templates/init/guided_aws/ClientBuilder.java

```diff
@@ -0,0 +1,29 @@
+package {{ package_name }};
+
+import software.amazon.awssdk.core.SdkClient;
+// TODO: replace all usage of SdkClient with your service client type, e.g; YourServiceAsyncClient
+// import software.amazon.awssdk.services.yourservice.YourServiceAsyncClient;
+// import software.amazon.cloudformation.LambdaWrapper;
+
+public class ClientBuilder {
+  /*
+  TODO: uncomment the following, replacing YourServiceAsyncClient with your service client name
+  It is recommended to use static HTTP client so less memory is consumed
+  e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/ClientBuilder.java#L9
+
+  private static class LazyHolder {
```

Comment:
Why we need an extra static class?  The SERVICE_CLIENT could be static for the `ClientBuilder` itself.